docs/spec/v2beta1/helmreleases.md

```diff
@@ -215,6 +215,10 @@ type HelmChartTemplateSpec struct {
 	// +optional
 	// +deprecated
 	ValuesFile string `json:"valuesFile,omitempty"`
+
+	// IgnoreMissingValuesFiles controls whether to silently ignore missing values files rather than failing.
+	// +optional
+	IgnoreMissingValuesFiles bool `json:"ignoreMissingValuesFiles,omitempty"`
```

Comment:
Undo this change, v2beta1 has no such field.

Reply:
Fixed. I'd originally targeted both v2beta1 and v2beta2, but anticipated that, as for source-controller, the beta1 API was deprecated and shouldn't be updated. I just forgot to revert this :-)